config.m4

```diff
@@ -20,7 +20,6 @@ if test "$PHP_XDEBUG" != "no"; then
   CPPFLAGS="$INCLUDES $CPPFLAGS"
 
   AC_CHECK_FUNCS(gettimeofday)
-  AC_CHECK_HEADERS([sys/select.h])
```

Comment:
Sorry - I missed epoll vs poll here. I didn't spot a WSAPoll though for Windows until after I saw the macro that defines it to poll though. I think I'm fine for supporting only Vista/2008 from Xdebug 2.5 onwards.